### PR TITLE
Return request to join acceptance notifications

### LINKED ISSUE #17955: Sometimes toast about old join community notification starts appearing on every login
There are no reproduction steps yet. Hope attached logs might help. 

For now I have faced the issue only on Android but I don't think it is device specific. It is hard to verify as it happens randomly.

On the video below (recorded on release build) you can see the toast appearing on every relogin. 

[Status-debug-logs (83).zip](https://github.com/status-im/status-mobile/files/13426193/Status-debug-logs.83.zip)

https://github.com/status-im/status-mobile/assets/97245802/bd716afe-e0fe-4bab-80d9-addf26bf1587

### Additional Information

- Status version: release 1.26, nightly
- Operating System: Android

### PR DESCRIPTION
**Pull request by ibrkhalil**
Returns a notification on the messenger response when joining a community.

Closes https://github.com/status-im/status-mobile/issues/17955 on status-mobile

### CODE REVIEW

**Review by cammellos — APPROVED** [review]

**Comment by pavloburykh** [comment]
Hey @ibrkhalil! Is PR ready for QA? I see it has been approved but still does not have request manual qa lablel and remains in Review column.

**Comment by pavloburykh** [comment]
Hey @ibrkhalil! Is there a corresponding mobile PR to test the fix or it is still in progress?

**Comment by ibrkhalil** [comment]
> Hey @ibrkhalil! Is there a corresponding mobile PR to test the fix or it is still in progress?

Yes Pavlo, It's this
https://github.com/status-im/status-mobile/pull/19162

File: protocol/messenger_communities.go [hunk 1]
```diff
@@ -1736,7 +1750,14 @@ func (m *Messenger) LeaveCommunity(communityID types.HexBytes) (*MessengerRespon
 	if err != nil {
 		return nil, err
 	}
+	mr.AddActivityCenterNotifications(notifications)
```
Comment (cammellos) [inline protocol/messenger_communities.go]: not sure I understand this code, why have we changed delete/dismiss, and why are we repulling and readding all notifications?

Comment (ibrkhalil) [reply protocol/messenger_communities.go]: It was an attempt to delete all of the notifications related to the community on leaving.
Anyways, I am drafting this now as we decided to delay working on this for now.

Comment (cammellos) [reply protocol/messenger_communities.go]: :+1
I think we should start testing the only change we have made before, and take it from there

**Comment by siddarthkay** [comment]
@ibrkhalil : pls rebase and iOS stage should be fixed in CI

**Pull request merged**